neato/vacuum.py

```diff
@@ -191,15 +191,15 @@ def update(self):
         if self._robot_has_map:
             robot_map_id = {}
             if self._state['availableServices']['maps'] != "basic-1":
-                if 'id' in self._robot_maps[self._robot_serial][0]:
+                if self._robot_maps[self._robot_serial]:
```

Comment:
Would this still check to see if there is data after the robots serial?  The json still returns the robot serial but the dict after it would be empty.

Reply:
This will be true if the render of ` self._robot_maps[self._robot_serial]` return a value.
`[]` would not be considered a value.